code/game/objects/items/storage/backpack.dm

```diff
@@ -241,6 +241,57 @@
 	desc = "A spacious backpack with lots of pockets, worn by Engineers of an Emergency Response Team."
 	icon_state = "ert_engineering"
 
+/obj/item/storage/backpack/engineerborg
+	name = "Engineering Cyborg bag"
+	desc = "Store the tools to repair that breach in this cute borgiebag!"
+	icon_state = "engiborgpack"
+	resistance_flags = FIRE_PROOF
+
+/obj/item/storage/backpack/miningborg
+	name = "Mining Cyborg bag"
+	desc = "Store the parts to repair your mechanical mining friends in this tough bag! Ash dust emanates from it constantly"
+	icon_state = "mineborgpack"
+
+/obj/item/storage/backpack/janitorborg
+	name = "Janitor Cyborg bag"
+	desc = "Clean the halls in style with this adorable borgbag! It smells strongly of cleaning products."
+	icon_state = "janiborgpack"
+
+/obj/item/storage/backpack/syndiassaultborg
+	name = "Suspicious Assault Cyborg bag"
+	desc = "Store the bullet to bring down the entirety of Nanotrasen with the comfort of a king in this adorable backpack! It gives off an odor of fresh blood and brass."
+	icon_state = "syndiassaultborgpack"
+
+/obj/item/storage/backpack/syndimedborg
+	name = "Suspicious Medical Cyborg bag"
+	desc = "Heal our brave operatives with medicines stored in this adorable bag! It gives off a scent of Omnizine."
+	icon_state = "syndimediborgpack"
+
+/obj/item/storage/backpack/syndiengborg
+	name = "Suspicious Engineering Cyborg bag"
+	desc = "Store the tools that will lead to the demise of Nanotrasen in an adorable bag! A smell of omnizine infuzed oil emanates from it."
```

Comment:
Seems the others use ``this`` instead of an. 
I can see some of the others mix and match it, so it's your call.
```suggestion
	desc = "Store the tools that will lead to the demise of Nanotrasen in this adorable bag! A smell of omnizine infuzed oil emanates from it."
```